workers/worker_git_integration.py

```diff
@@ -1001,9 +1001,12 @@ def update_gh_rate_limit(self, response, bad_credentials=False, temporarily_disa
                 self.oauths[0]['rate_limit'] -= 1
                 self.logger.info("Headers did not work, had to decrement")
                 time.sleep(30)
+
         self.logger.info(
             f"Updated rate limit, you have: {self.oauths[0]['rate_limit']} requests remaining."
         )
+
+        #Stalls after here for some reason.
```

Comment:
I have seen stalling on heavily taxed machines. Are you seeing this only in the facade worker resolution or elsewhere?

Reply:
I haven't seen it elsewhere yet and it hasn't happened with facade when I have been testing it most recently. I think that comment might be talking about a bug that has since been squashed.